dev/benchmarks/macrobenchmarks/lib/src/color_filter_with_unstable_child.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class ColorFilterWithUnstableChildPage extends StatefulWidget {
+  const ColorFilterWithUnstableChildPage({super.key});
+
+  @override
+  State<StatefulWidget> createState() => _ColorFilterWithUnstableChildPageState();
+}
+
+class _ColorFilterWithUnstableChildPageState extends State<ColorFilterWithUnstableChildPage> with SingleTickerProviderStateMixin {
+  late Animation<double> _offsetY;
+  late AnimationController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = AnimationController(vsync: this, duration: const Duration(seconds: 2));
+    // Animations are typically implemented using the AnimatedBuilder widget.
```

Comment:
This benchmark has no history to maintain. It looks like this comment and implementation were cut/pasted from some other benchmark that had to deal with consistency, but that doesn't apply here. Shouldn't it use the more recommended mechanism in this case?

Reply:
Done, using AnimatedBuilder now.